Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -36,7 +37,8 @@ namespace species
         return PhysicalSpecies::unspecified;
     }
 
-    static amrex::Real get_charge (PhysicalSpecies ps)
+    AMREX_FORCE_INLINE
+    amrex::Real get_charge (PhysicalSpecies ps)
```

Comment:
Fixes the warning:
`./Source/Particles/SpeciesPhysicalProperties.H:39:24`: warning: ‘amrex::Real species::get_charge(PhysicalSpecies)’ defined but not used [-Wunused-function]
     static amrex::Real get_charge (PhysicalSpecies ps)